packages/mui-joy/src/CardOverflow/CardOverflow.tsx

```diff
@@ -44,67 +46,61 @@ const CardOverflowRoot = styled('div', {
   const childRadius = 'calc(var(--CardOverflow-radius) - var(--variant-borderWidth, 0px))';
   return {
     alignSelf: 'stretch', // prevent shrinking if parent's align-items is not initial
-    borderRadius: 'var(--CardOverflow-radius)',
     position: 'relative',
     display: 'flex',
-    ...(ownerState['data-parent'] === 'Card-horizontal' && {
+    margin: 'var(--_CardOverflow-margin)',
+    borderRadius: 'var(--_CardOverflow-radius)',
+    padding: 'var(--_CardOverflow-padding)',
+    [`.${cardClasses.horizontal} > &`]: {
```

Comment:
When is this class being set? Isn't this a parent classname?

Reply:
Do we expect people to set it themselves?

Reply:
> When is this class being set?

It is set by the Card component. The CardOverflow should be used as a direct child of the Card.

> Do we expect people to set it themselves?

Nope, they don't need to.

Reply:
Aaah, we are basically using a parent selector in a way, nicee :D I never thought of using this :)